### DISCUSSION TITLE
モブアイディア

### DISCUSSION THREAD

**Discussion by Suyarumi** [TUSB/TheUnusualSkyBlock#91] [Idea]
モブのアイディアを記録する場所です。
記録するアイディアは決まってない箇所があっても問題ありません。

**Comment by Suyarumi** [comment]
アイディア
名前:未定
見た目:未定
タイプ:雑魚モブ
スポーンする場所:未定
説明:プレイヤーを見つけるとそのプレイヤーに化け、装備とMPをコピーする。
またコピーしたプレイヤーの職業によってこのモブの使用するスキルが変化する。

**Comment by Suyarumi** [comment]
アイディア
名前:未定
見た目:スケルトン
タイプ:雑魚モブ
スポーンする場所:序盤の島
説明:矢を10本くらい散弾銃のように打ってくるスケルトン。

**Comment by Suyarumi** [comment]
アイディア
名前:未定
見た目:頭以外は青色の革防具で頭に数字の書いてあるプレイヤーヘッドを被った人型モブ
タイプ:ボス
説明:
最初は頭のカウントが9だが
倒すたびに頭の数字が1ずつ減っていきカウントが0になると倒せる。
カウントが変わるたびにスキルやステータスなどの性能が変わっていく。
カウントごとの性能
9:
8:杖(にんじん棒)を装備して戦う。魔法防御力は高め。魔法を主に使う。魔法は炎、氷、雷、光、闇、回復を使う。幻影も使用する。MPが低くなってくると周りのプレイヤーから奪う。
7:6体の分身を召喚し、7体で戦う。その代わりこの形態のHPは低め。
6:剣を装備して戦う。移動速度が速く、HPは低め、短距離ワープをしたり、一瞬だけ盲目付与したり、透明化を行うことでプレイヤーを撹乱しながら戦う
5:弓を使用して戦う。弓を打つ間隔はかなり短い。近づかれたらバックステップしつつ強力な一撃を放つ。罠を設置してデバフを与える。
4:盾を装備して戦う。防御力が高くHPは低め、自分から攻撃はしてこないがダメージを受けると発動するスキルを多く持つ。
3:斧を装備して戦う。攻撃力とHPが高く、全体的に隙が多い。
2:トライデントを使って戦う。リーチがちょっと長い。スキルもリーチが長めの攻撃を中心に行う。
1:剣と盾を装備して戦う。HPが自動で回復する。HP、攻撃力、防御力、移動速度がそこそこ高め、HPが低くなるほどスキル使用間隔が早くなり、攻撃力、防御力が上がっていく。死んでも一度だけステータスが上昇して生き返り、大技を放ってくる。
0:鎌(クワ)を装備して戦う。HPは最も高い。自分と同じ見た目の雑魚を召喚しながら戦う。召喚された雑魚モブはそれまでの形態のスキルを使用する。

**Comment by Suyarumi** [comment]
アイディア
名前:未定
見た目:花火の上にクリーパーが乗っている。
スポーンする場所:クリーパー島
説明:クロスボウ追加により花火を自由自在に飛ばすことができるようになったので
追尾機能をつけた花火の上にクリーパーを乗せる。
花火の爆発と同時にクリーパーも爆発する。

**Comment by Suyarumi** [comment]
アイディア
名前:魚人
見た目:魚っぽいスキンの頭をかぶったドラウンド
スポーンする場所:テーブルマウンテン1~3F
説明:能力などは未定

**Comment by Suyarumi** [comment]
アイディア
名前:アングリーゴーレム
見た目:頭付近からAngryVillagerパーティクル出してるアイアンゴーレム。
スポーンする場所:テーブルマウンテン5~6F
説明:主に通路を塞ぐように配置して、耐久力も高いお邪魔モブっぽい立ち位置にする。

**Comment by Suyarumi** [comment]
アイディア
名前:ヘルハウンド
見た目:狼
スポーンする場所:ネザー
説明:群れでスポーンする。
時々に遠吠えを行い遠吠えをすると近くにいるヘルハウンドも一緒に遠吠えを行い、遠吠えした数が多いほど攻撃力と防御力を強化する。　要するに群れるほど強くなる。

**Comment by Suyarumi** [comment]
アイディア
名前:未定
見た目:未定
スポーンする場所:未定
説明:周囲の敵を死亡時に爆発させるようにする。

**Comment by Suyarumi** [comment]
アイディア
名前:未定
見た目:クロスボウ持ったピグリン
スポーンする場所:テーブルマウンテン橋にある塔
説明:移動速度を持たないが、遠距離から花火で狙撃してくる。30秒くらいで消滅する。

**Comment by Suyarumi** [comment]
アイディア
名前:ミミック
見た目:未定
スポーンする場所:補給物資、テーブルマウンテン
タイプ:雑魚モブ
説明:未定

**Comment by Suyarumi** [comment]
アイディア
名前:未定
見た目:毒々しいパーティクルを纏ったミツバチ
スポーンする場所:地下世界
タイプ:雑魚モブ
説明:攻撃されると毒をもらう。倒すと破裂して毒をばら撒く。

**Comment by Suyarumi** [comment]
アイディア
名前:未定
見た目:ブレイズ
スポーンする場所:ネザー
タイプ:雑魚モブ
説明:超強化ブレイズ、プレイヤーの周りを機敏に飛び回りながら着弾で爆発するレーザーを打つ。

**Comment by Suyarumi** [comment]
アイディア
名前;ウィザーマンサー
見た目:石のクワを持ったウィザースケルトン
スポーンする場所:ネザーのSSの場所
![2021-02-16_19 52 53](https://user-images.githubusercontent.com/43696755/108053390-b2111e80-7090-11eb-8156-60c8c925149c.png)
説明:強化されたハスクを召喚する。あとは未定

**Comment by Suyarumi** [comment]
アイディア
名前:生き貯金豚
見た目:豚の上に金のヘルメットを被ったスケルトンが乗っている。
スポーンする場所:テーブルマウンテン
タイプ:雑魚モブ
説明:たまにスポーンする。倒すとたくさんエメラルドをドロップする。
シートにあったアイディアを使用。

**Comment by Suyarumi** [comment]
アイディア
名前:未定
見た目:雷
スポーンする場所:未定
タイプ:雑魚モブ
説明:プレイヤーに向かって移動してくる雷のように見えるモブ。
実際は透明なシルバーフィッシュに雷を落とし続ける。

**Comment by Suyarumi** [comment]
アイディア
名前:デュラハン
見た目:頭以外は鉄防具を着ていて、手に剣とゾンビの頭を持った透明なゾンビ。
スポーンする場所:未定
タイプ:雑魚モブ
説明:プレイヤーに向けて自分の頭を投げてくる。
投げた頭は以後自立行動するようになる。
自立行動するようになった頭はプレイヤーに噛み付いてくる。

**Comment by Suyarumi** [comment]
アイディア
名前:未定
見た目:頭以外は金防具を着ていて、手に剣と一つ目の頭を持った透明なゾンビ
スポーンする場所:未定
タイプ:雑魚モブ
説明:デュラハンの上位種
プレイヤーに向けて自分の頭を投げてくる。
投げた頭は以後自立行動するようになり、プレイヤーと距離を取りながらレーザーを打つようになる。

**Comment by Suyarumi** [comment]
アイディア
名前:エンダーミミック
見た目:エンダーミミックっぽい見た目
スポーンする場所:補給物資とかテーブルマウンテンとか
タイプ:雑魚モブ
説明:補給物資からたまに出てくるのを想定。
倒すと報酬を落とす。
プレイヤーの周りに連続でテレポートし、テレポート場所に自身の幻影を設置する。
幻影はテレポート終了後に動き出す。
幻影は10秒で消える。
幻影に攻撃されるとMPを吸収される。

**Comment by Suyarumi** [comment]
アイディア
名前:未定
見た目:シュルカー
スポーンする場所:エンダー島
説明:一度に3つシュルカーの弾を放ってくる。

**Comment by Suyarumi** [comment]
アイディア
名前:ガーディアン
見た目:ゴーレム
スポーンする場所:テーブルマウンテン橋
説明:移動速度は遅い。
下にあるブロックを投げて攻撃してくる。(地形破壊なし)

**Comment by Suyarumi** [comment]
アイディア
名前:雪男
見た目:未定
スポーンする場所:トカルトコルデ
説明:地面のブロックを複数投げて攻撃する。(地形破壊なし)

**Comment by Suyarumi** [comment]
アイディア
名前:デリシャスフルーツ
見た目:りんご
スポーンする場所:テーブルマウンテン橋にある木
説明:プレイヤーに向かって飛んでいき、破裂する。

**Comment by Suyarumi** [comment]
アイディア
名前:未定
見た目:大木
スポーンする場所:未定
タイプ:ボス
説明:実を周囲にばら撒き、そこから木を生やす。
生えてくる木には種類があり、行動パターンが違う。
1:成長すると根っこを周囲のプレイヤーに伸ばしてから体力とMPを奪い続ける。
2:成長すると周囲に葉っぱをナイフのように飛ばして攻撃する。
3:成長すると周囲にいがぐりをばらまきし、踏んだプレイヤーにダメージを与える。

**Comment by Suyarumi** [comment]
アイディア
名前:トカルトコルデの魔術師
見た目:ウィッチ
スポーンする場所:トカルトコルデ
タイプ:雑魚モブ
説明:トカルトコルデの魔術師なので氷の魔法を多用する。
氷で氷柱を周囲に作り、落とす。落ちたときに衝撃が発生する。
氷の竜巻を起こして、入ったプレイヤーを凍結させる。
動く氷像を作り、プレイヤーを攻撃させる。氷像は壊れてもすぐに再生される。
氷の武器を作り続け、プレイヤーに飛ばしたりして攻撃をする。

**Comment by Suyarumi** [comment]
アイディア
名前:ムーンエンペラー
見た目:未定
スポーンする場所:月
タイプ:ボス
説明:スポーンすると死ぬまで流星群を出し続ける
流星群は自機狙いとランダムに落ちるものが存在する。

**Comment by Suyarumi** [comment]
アイディア
名前:アグレッシブタートル
見た目:亀
スポーンする場所:テーブルマウンテン1~3F
タイプ:雑魚モブ
説明:攻撃してくるタイプの亀
耐久と攻撃力は高めに設定する。

**Comment by Suyarumi** [comment]
アイディア
名前:スレイヤー
見た目:忍者の職業モブ
スポーンする場所:未定
タイプ:雑魚モブ
説明:忍者なのでカラテを使ってくる。
行動パターン
奇襲攻撃:通常攻撃とは別に行う攻撃。ダメージが非常に高い。
隠密:近くにプレイヤーがいないときに使用。使用すると装備も含めて完全に透明化し、落下ダメージも無効化する。ダメージを受けると解除される。プレイヤーが4ブロック以内にいてかつプレイヤーが視線を外している場合に透明化を解除し、奇襲攻撃を行う。隠密中は他のスキルを使わなくなる。
スリケン:スリケンを投げる。プレイヤーが近くにいるときはバックステップも行う。
煙幕:煙幕を投げてプレイヤーの視界を塞ぎ、盲目をかけ続ける。
幻影:
土遁:地面に潜ってプレイヤーの後ろから出現して奇襲攻撃を行う。
潜ってるときはパーティクルが出る。
サヨナラ:死亡時に発動し、サヨナラ！と発言した1秒後に爆発する。
行動順
隠密→煙幕→スリケン→土遁→スリケン

**Comment by Suyarumi** [comment]
アイディア
名前:黄金キノコ
見た目:未定
タイプ:雑魚モブ
説明:倒すと黄金キノコという素材を落とす。
基本的にプレイヤーから逃げるように行動する。
近づかれたら、溶けた金をプレイヤーに向かって吐く。
溶けた金に当たると炎上+火傷+大ダメージ。

**Comment by Suyarumi** [comment]
アイディア
名前:レインボーシープ
見た目:_jebって名前のついた羊の上に名前のついたAECを乗せる。
タイプ:雑魚モブ
説明:体力がとても高く　防御力も高い。
プレイヤーからとんでもない速度で逃げる上、短距離ワープ持ち。
プレイヤーに近づかれると、プレイヤーの視点を180度回転させ、その方向に自身の偽物を出現させる。
発動時にワープ音とそれっぽいパーティクルを出す。

**Comment by anminmakura** [comment]
アイディア
名前:穏やかな心を持ちながら激しい怒りによって目覚めた木
見た目:オークの木
タイプ:雑魚モブ
スポーンする場所:特定の地点の木を伐採すると出現
説明:プレイヤーが木を伐採すると同時にスポーンする。
単純な殴り特化系で、自身の高いHPと回復スキルを使用して長期戦を強いられるタイプのMOB。
プレイヤーが距離を取ると遠距離攻撃でリンゴを発射するようになる。
弱点として炎上させると即死する性質がある。

**Comment by Suyarumi** [comment]
アイディア
名前:キラーキャット
タイプ:雑魚モブ
スポーンする場所:未定
説明:プレイヤーから一定距離を保ち続ける。
超低速のふれると大爆発する空気弾を飛ばしてくる。
空気弾はプレイヤーを追跡し、壁を抜け、とても見えにくい。

**Comment by Suyarumi** [comment]
アイディア
名前:superbia
見た目:
スポーンする場所:深淵様が召喚
タイプ:雑魚モブ
説明:
傲慢の名を冠する。
攻撃力はかなり低い。
味方を指揮することで強化をし、自分の周り15ブロックのプレイヤーにデバフを与える。

名前:avaritia
見た目:全能ノ一を両手に持っている。
スポーンする場所:深淵様が召喚
タイプ:雑魚モブ
説明:
強欲の名を冠する。
全能ノ一を投げて攻撃してくる。
投げ方は2種類
縦回転で直線的に投げてくる。
横回転でブーメランみたいに投げてくる。
攻撃力が非常に高い。

名前:invidia
見た目:
スポーンする場所:深淵様が召喚
タイプ:雑魚モブ
説明:
嫉妬の名を冠する。
自分がターゲットしたプレイヤーが自分より上にいると、影のようなものをプレイヤーに向けて伸ばして攻撃してくる。
ターゲットしたプレイヤーがエリトラで滑空していた場合は伸ばす速度が3倍くらいになり、当たった場合は滑空状態を解除する。

名前:ira
見た目:
スポーンする場所:深淵様が召喚
タイプ:雑魚モブ
説明:
憤怒の名を冠する。
時間経過で自身のLvが上昇していく。
炎を飛ばして攻撃を行う。
プレイヤーに向けて3発炎弾を飛ばす。
自身の周りを囲うように火柱を立たせる。
無差別に炎弾をばらまき、地面を炎上させる。

名前:luxuria
見た目:
スポーンする場所:深淵様が召喚
タイプ:雑魚モブ
説明:
色欲の名を冠する。
周囲のプレイヤーを魅了し、視線を集中させる。

名前:gula
見た目:手に何かしら食べ物を持っている。
スポーンする場所:深淵様が召喚
タイプ:雑魚モブ
説明:
暴食の名を冠する。
HPが少なくなるといったん逃げ、手に持っている食べ物を食べて回復する。
回復量は手の持っている食べ物によって変わる。
手に持つ食べ物の種類と効果
パン(HP25%回復)
焼き魚(HP30%回復)
ステーキ(HP50%回復)
ケーキ(HP70%回復)
上位金リンゴ(HP全回復)
TNT(自爆)

名前:acedia
見た目:
スポーンする場所:深淵様が召喚
タイプ:雑魚モブ
説明:
怠惰の名を冠する。
HPが異常に高く(倒させる気がないレベル)
攻撃力も高いが...?
行動パターン
召喚直後:自分の前方に竜巻を複数出す。竜巻に触れてもダメージは受けず打ち上げられるだけである。
2秒経過:プレイヤーに向けて氷の柱を伸ばす、プレイヤーがこの攻撃を受けると拘束される。この際竜巻が氷を纏うようになり、ダメージを受けるようになる。
4秒経過:プレイヤーに向けて火柱を3wayの立たせる。この際竜巻が炎も纏うようになり、ダメージが上昇し、触れると火傷を受けるようになる。
6秒経過:自分の周り半径3ブロックに雷を落としまくる。この際竜巻が雷を纏うようになり、ダメージが上昇し、触れると移動速度低下を受けるようになる。
8秒経過:光弾をプレイヤーに向けて大量に飛ばす。当たるとダメージと一瞬だけ盲目が付与される。この際竜巻が光を纏うようになり、光の矢を周囲にばらまくようになる。
9秒経過:竜巻が消滅する。
10秒経過:帰宅する。

**Comment by Suyarumi** [comment]
名前:未定
見た目:色が金属っぽい深淵様
スポーンする場所:砂漠
タイプ:ボス
説明:深淵様をモデルにしたロボット。
人が作ったものなので深淵様よりは弱い。
